extra_stuff/lessons_in_progress/git2/git_branching.md

```diff
@@ -0,0 +1,60 @@
+# Git Branching
+
+Using Git Branch enables you to take your development path into new directions. Branching gives you the ability to represent what is typically a linear and flat development process as a multi-dimensional mind-map.
```

Comment:
I think we are overcomplicating the explanation of what a branch is. There are a lot of abstract concepts here. I think it would be better to explain that branches allow you to work on a new feature or experiment safely without affecting the main master codebase.